spec/linter-xo-spec.js

```diff
@@ -59,6 +59,15 @@ describe('xo provider for linter', () => {
 		});
 	});
 
+	describe('checks relative-path.js and', () => {
+		it('shows no error notifications', async () => {
```

Comment:
Does Jasmine have any way of marking the test as known failing?

Reply:
Jasmine has [`pending()`](https://jasmine.github.io/api/3.7/global.html#pending)

> Mark a spec as pending, expectation results will be ignored.

However atom doesn't have it.. only has [`xdescribe()`](https://jasmine.github.io/api/3.7/global.html#xdescribe) and [`xit()`](https://jasmine.github.io/api/3.7/global.html#xit)..